lib/trento/domain/sap_system/sap_system.ex

```diff
@@ -28,8 +28,9 @@ defmodule Trento.Domain.SapSystem do
 
   That being said, this is the logical order of events in order to register a full system:
 
-  1. A SAP system discovery message with a new database instance is received. At this point, the
-     registration process starts and the database is registered.
+  1. A SAP system discovery message with a new database instance is received.
+     We ignore database with Secondary role in a system replicated scenarios.
```

Comment:
I would avoid using `we` term. `Database instances with Secondary role in a system replication scenario are discarded`

Reply:
done